### DISCUSSION TITLE
The "Always Start typing in English Mode" mode does not work.

### DISCUSSION THREAD

**Discussion by serggrod** [Flow-Launcher/Flow.Launcher#2423] [General]
The "Always Start typing in English Mode" mode does not work. Despite the fact that this mode is enabled and it’s just a super feature for me Unfortunately it doesn't work on my Windows 10. Perhaps you can tell me what I'm doing wrong. Or maybe you’ll just find out that it doesn’t work and fix something

**Comment by jjw24** [comment]
Hi there what language is your OS and from what language are you switching to the "Always Start typing in English Mode" mode ?

**Comment by serggrod** [comment]
Hi! I switch from Russian or Ukrainian, the effect is the same, switching
does not work

ср, 22 нояб. 2023 г. в 12:17, Jeremy Wu ***@***.***>:

> Hi there what language is your OS and from what language are you switching
> to the "Always Start typing in English Mode" mode ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Flow-Launcher/Flow.Launcher/discussions/2423#discussioncomment-7640550>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACH4TUAUFB5RMC24ORR7BODYFXGM5AVCNFSM6AAAAAA7N6O522VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMNBQGU2TA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 
С Уважением
Сергей Гродецкий
Региональный Менеджер,Юг и  Восток Украины
ООО Продторгресурс
тел. 0674808248

**Comment by VictoriousRaptor** [comment]
See https://github.com/Flow-Launcher/Flow.Launcher/issues/2108

**Comment by serggrod** [comment]
Thanks for the tip!

ср, 22 нояб. 2023 г. в 15:25, VictoriousRaptor ***@***.***>:

> See #2108 <https://github.com/Flow-Launcher/Flow.Launcher/issues/2108>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Flow-Launcher/Flow.Launcher/discussions/2423#discussioncomment-7642451>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACH4TUEXSRUQIK7WCUJMPYLYFX4LPAVCNFSM6AAAAAA7N6O522VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMNBSGQ2TC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 
С Уважением
Сергей Гродецкий
Региональный Менеджер,Юг и  Восток Украины
ООО Продторгресурс
тел. 0674808248
